src/ast.ts

```diff
@@ -42,7 +45,7 @@ import {
 } from "./types";
 
 /** Indicates the kind of a node. */
-export enum NodeKind {
+export const enum NodeKind {
```

Comment:
In the early days, all enums were `const` which disabled reverse lookup in JS. That's sometimes useful in debugging, though, say to find out the value of `NodeKind[someNode.kind]`. So I eventually dropped this to aid debugging in favor of having all-`const` enums but disabling them again anyway with `preserveConstantEnums`, also because it doesn't make a difference when compiled to Wasm. There is also the concern that the same applies to transform users, who may have a better time if they can do reverse lookups to debug their transforms, so making two builds doesn't really help much either. Any ideas?

Reply:
Without this changes benefit will be not so significant. constants is quite important here

Reply:
Imo this is an OK price to pay for making the code base more approachable / have less surprises, also given that our focus will one day switch to Wasm builds anyway where it doesn't matter.